src/store/dposPlugin.ts

```diff
@@ -75,18 +97,6 @@ function observeState(store: Store<any>, stateGetter): Observable<any> {
 
 
 
-function buildLoadHistoryTrigger(store) {
-    combineLatest(
-        observeState(store, (state) => state.DPOS.web3),
-        observeState(store, (state) => state.DappChain.GatewayInstance),
```

Comment:
Since we're not using the gatewayInstance anymore, can we remove it from the `dappchainStore.ts`?